lib/AST/Attr.cpp

```diff
@@ -601,7 +601,9 @@ StringRef DeclAttribute::getAttrName() const {
         return "effects(readnone)";
       case EffectsKind::ReadOnly:
         return "effects(readonly)";
-      case EffectsKind::ReadWrite:
+      case EffectsKind::ReleaseNone:
+        return "effects(releasenone)";
+     case EffectsKind::ReadWrite:
```

Comment:
Unintentional outdent?

Reply:
Thanks I have fixed this in a later commit.